lib/security/HttpBasicAuthenticator.js

```diff
@@ -68,11 +79,14 @@ module.exports = oo({
 
   /*****************************************************************************
    * @method validateCreds
-   * @description validateCreds description
-   * @param {string} username -- xxx
-   * @param {string} password -- xxx
-   * @throws {Service.errors.InternalServerError} -- xxx
-   * @returns {xxx} -- xxx
+   * @description Finds a user matching a username and password. The password is
+   *              checked using the hash function.
+   * @param {string} username -- username from the HTTP request
+   * @param {string} password -- password from the HTTP request
+   * @throws {Service.errors.InternalServerError} -- 500 Internal Server Error
+   * @returns {object|undefined} -- Object representing the user if a user matching
```

Comment:
the six primitive types should be lowercased (boolean, number, string, undefined, null, and symbol). everything else should be uppercased.